app/controllers/JourneyAnswersController.scala

```diff
@@ -133,6 +133,25 @@ class JourneyAnswersController @Inject() (
       }
     }
 
+  def savePropertyRentalsAndRentARoomAbout(
+    taxYear: TaxYear,
+    incomeSourceId: IncomeSourceId,
+    nino: Nino
+  ): Action[AnyContent] =
+    auth.async { implicit request =>
+      withJourneyContextAndEntity[RaRAbout](
+        taxYear,
+        incomeSourceId,
+        nino,
+        JourneyName.PropertyRentalsAndRentARoomAbout,
+        request
+      ) { (ctx, rarAbout) =>
+        handleResponse(CREATED) {
+          propertyService.saveRaRAbout(ctx, nino, rarAbout)
```

Comment:
`RaRAbout` Are we not creating separate case class? we need to capture more fields

Reply:
Done. Thanks